module/zfs/spa.c

```diff
@@ -3375,6 +3376,12 @@ spa_activity_check(spa_t *spa, uberblock_t *ub, nvlist_t *config)
 
 	import_expire = gethrtime() + import_delay;
 
+	char buf[128];
+	(void) snprintf(buf, sizeof (buf), "Checking MMP activity, waiting "
+	    "%llu ms", (u_longlong_t)NSEC2MSEC(import_delay));
+	spa_import_progress_set_notes(spa, buf);
```

Comment:
I think this and below could be cleaner if spa_import_progress_set_notes() be printf-like, taking format and variable arguments.

Reply:
Good idea.  There's another case for the spa_log_spacemap notes.  I'll make the change.